ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml

```diff
@@ -36,6 +36,7 @@ spec:
   workspaces:
     - name: output
       description: Storage for preflight results, logs and artifacts.
+      optional: true
```

Comment:
Why is that?

Reply:
While running the hosted pipeline, here the preflight task was failing with error: output file not found, so I make it optional.

Reply:
It's because the workspace is not added on the level of the pipeline. Could you add it there instead?

Reply:
sure